AUTHORS

```diff
@@ -0,0 +1,8 @@
+BeagleTerm was originally created in Oct 2012. Below is an list of the
+project's contributors -- people who have submitted patches, and have
+helped to make BeagleTerm a better product.
+
+Sungguk Lim <sungguk.lim@gmail.com>
```

Comment:
Well since it's your project I wanted to list your name first. Everyone else is listed based on number of commits. I figured going forward people could just add to the bottom of the list. Otherwise we'd have to reorganize the file each time.